db/queries.js

```diff
@@ -0,0 +1,24 @@
+const connection = require('../knexfile')['development'];
+const database = require('knex')(connection);
+
+module.exports = {
+    getAllBooks() {
+      return database('books');
+    },
+    getSingleBook(request) {
+      return database('books').where('id', request.params.id).select();
+    },
+    addBookToFavs(favorite) {
+      return database('favorites').insert(favorite, 'id');
+    },
+    removeBookFromFavs(request) {
+      return database('favorites').where('isbn', request.params.isbn)
+        .del()
+        .then(() => {
```

Comment:
The .then that held us up forever! We should ask Robbie more about why this was needed when we have our standup with him.